includes/class-wc-gateway-stripe.php

```diff
@@ -961,4 +969,35 @@ public function get_intent_object( $intent_id = '' ) {
 
 		return $intent_object;
 	}
+
+	/**
+	 * Displays a notice that 3DS is not a setting anymore.
+	 *
+	 * @since 4.2.0
+	 */
+	public function display_three_d_secure_notice() {
+		if ( $this->get_option( '3ds_setting_notice_dismissed' ) ) {
+			return;
+		}
+
+		?>
+		<div data-nonce="<?php echo esc_attr( wp_create_nonce( 'no-3ds' ) ) ?>" class="notice notice-warning is-dismissible wc-stripe-3ds-missing">
+			<p>
+				<?php
+				// ToDo: Replace this URL with a Stripe link or a WooCommerce article.
+				$url     = 'http://example.com';
```

Comment:
I think https://stripe.com/docs/payments/dynamic-3ds would be a good URL to point to.